README.md

```diff
@@ -0,0 +1,3 @@
+# .github
```

Comment:
Personal opinon: I would remove this file since it's not really adding any value

Reply:
It adds this nice doc

![image](https://github.com/NordSecurity/.github/assets/73230019/b15066ce-13c2-4c9b-966f-b464ede31e3b)

but sure I can omit it.

Reply:
I know what the README does, I just don't see the point in having it if it doesn't add any valuable information